src/js/gi/selectors/calculator.js

```diff
@@ -183,15 +177,18 @@ const getDerivedValues = createSelector(
     } else if (isCorrespondence) {
       tuitionFeesCap = constant.CORRESPONDTFCAP;
     } else if (isPublic && institution.country === 'usa') {
-      tuitionFeesCap = inputs.inState
+      tuitionFeesCap = inputs.inState === 'yes'
                      ? institution.tuitionInState
                      : institution.tuitionOutOfState;
     } else if (isPrivate || isForeign) {
       tuitionFeesCap = constant.TFCAP;
+    } else {
+      // TODO: What should default cap be?
```

Comment:
The TFCAP for "For Profit" schools is the same as private and foreign schools. 

That should be covered in the isprivate function.